tasks/sep/020-holocene-system-config-upgrade-multi-chain/VALIDATION.md

```diff
@@ -0,0 +1,150 @@
+      - The address of the Council Safe: `0xf64bc17485f0B4Ea5F06A96514182FC4cB561977`
+      - The safe hash to approve: `0x7e8055d58462ab08d75766766252966eda91b23097f8d96aca0547fe7aae078a`
+    - The using `cast index`, we can verify that:
+      ```shell
+        $ cast index address 0xf64bc17485f0B4Ea5F06A96514182FC4cB561977 8
+        0x56362ae34e37f50105bd722d564a267a69bbc15ede4cb7136e81afd747b41c4d
+        ```
+        and
+      ```shell
+        $ cast index bytes32 0x7e8055d58462ab08d75766766252966eda91b23097f8d96aca0547fe7aae078a 0x56362ae34e37f50105bd722d564a267a69bbc15ede4cb7136e81afd747b41c4d
+        0x3481a62ac310eecec9b2bcbdfc7f9759c1641b33ec9f302e19c8dc75aa3427bb
+        ```
+      And so the output of the second command matches the key above.
+
+### For the Foundation:
+
+- **Key:** `0x66833911cd4988ff9068991368a392dfd91753075a1080eee9ac5b6bf6a4815b` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000`<br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+  **Meaning:** The GnosisSafe `approvedHashes` mapping is updated to indicate approval of this transaction by the council. The correctness of this slot can be verified as follows:
+    - Since this is a nested mapping, we need to use `cast index` twice to confirm that this is the correct slot. The inputs needed are:
+      - The location (`8`) of the `approvedHashes` mapping in the [GnosisSafe storage layout](https://github.com/safe-global/safe-contracts/blob/v1.4.0/contracts/libraries/SafeStorage.sol#L23)
+      - The address of the Foundation Safe: `0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B`
+      - The safe hash to approve: `0x7e8055d58462ab08d75766766252966eda91b23097f8d96aca0547fe7aae078a`
+    - The using `cast index`, we can verify that:
+      ```shell
+        $ cast index address 0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B 8
+        0xc18fefc0a6b81265cf06017c3f1f91c040dc3227321d73c608cfbcf1c5253e5c
+      ```
+      and
+      ```shell
+        $ cast index bytes32 0x7e8055d58462ab08d75766766252966eda91b23097f8d96aca0547fe7aae078a 0xc18fefc0a6b81265cf06017c3f1f91c040dc3227321d73c608cfbcf1c5253e5c
+        0x66833911cd4988ff9068991368a392dfd91753075a1080eee9ac5b6bf6a4815b
+      ```
+      And so the output of the second command matches the key above.
+
+### Liveness Guard
+**Key:** `0x4a204f620c8c5ccdca3fd54d003badd85ba500436a431f0cbda4f558c93c34c8` <br/>
+**Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+**After:** `0x000000000000000000000000c26977310bc89daee5823c2e2a73195e85382cc7` <br/>
+**Meaning:** The `LivenessGuard` address is set to [0xc26977310bC89DAee5823C2e2a73195E85382cC7](https://sepolia.etherscan.io/address/0xc26977310bC89DAee5823C2e2a73195E85382cC7). The key can be validated by the key in the [Guard Manager](https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/base/GuardManager.sol#L30).
```

Comment:
I am not sure to understand this state overrides with the `livenessGuard`. 
I need to check this one before merging the PR, you got more infos about this change would be happy to have more infos.

Reply:
Ok done here [0726e80](https://github.com/ethereum-optimism/superchain-ops/pull/378/commits/0726e80299d23a023cfe0c20bc19f6a0adc86fdf). TBH I don't really understand this, I was just following instructions to copy paste from previous tasks.

Reply:
[be28707](https://github.com/ethereum-optimism/superchain-ops/pull/378/commits/be28707b4abc854a677c5e53cf1d08debb8fef0f)